_src/body/_body.jade

```diff
@@ -1,9 +1,12 @@
 .body__wrapper 
   .body__card
-    img(src="assets/dogola.png", alt="OOPSS")
-    h3.body__title Dogola, Rei dos Reis
-
-
+    .body__frame
+      img(src="assets/dogola.png", alt="OOPSS")
+    .body__border 
```

Comment:
There is no need for this div (.body__border) nor even its child. Instead we are going to approach it differently. Check out to the comment highlighting the .body__frame